packages/react-native-client/android/src/main/java/io/fishjam/reactnative/RNFishjamClient.kt

```diff
@@ -74,6 +75,8 @@ class RNFishjamClient(
       emitEvent(event, mapOf(event.name to value))
     }
 
+  private var foregroundServiceManager: ForegroundServiceManager? = null
```

Comment:
**question:** 
1. Can foregrund service be called when user is not connected to rom?
2. Does `foregroundServiceManager` must be nullable? From its curent use i think it can be not nullable

Reply:
1. Yes
2. How would that work? `configureForegroundService` must be called to instantiate a `ForegroundServiceManager` instance.

Reply:
Some default can be passed to initialize it in constructor or in property. Then when `configureForegroundService` is called manager would be updated.

Reply:
But this is my point of view. If sth can be not null I would prefer do it that way.

Reply:
It doesn't make sense. This is the reason there is a configure method. It will just move the null from `RNFishjamClient` to `ForegroundServiceManager` and we explicitly want to have configuration set once. It also provides additional info if the manager is already configured.

Reply:
It does not move null to `ForegraoundServiceManager`. Consider this code:
```suggestion
  class ForegroundServiceNotificationConfig : Record {
  @Field
  val channelId: String = "default_value"

  @Field
  val channelName: String = "default_value"
  @Field
  val notificationContent: String = "default_value"

  @Field
  val notificationTitle: String = "default_value"
}

  private var foregroundServiceManager: ForegroundServiceManager = ForegroundServiceManager(appContext!!, ForegroundServiceNotificationConfig()) 
```

In this case there is no null values, it could be directly used by developer without knowing what are specified properties in `ForegroundServiceNotificationConfig`, and only if user want to change it he would do it. Also `configureForegroundService` does not have to be called by user.

Reply:
> Also configureForegroundService does not have to be called by user
The user never calls it, it's our internal method that should be called once the hook is initialized. 

In my opinion, you're trying to fit a use case, to justify having a non-null value even when it makes perfect sense to have one. With your approach:
1. I'd have to create another method in the manager to reconfigure the config 
2. Create another variable to check if it was configured (it should not be changed once configured).

Reply:
1. I think that you can just create new manager (correct me if i am wrong)
2. Actually i think that notification text should be reconfigurable. User should be able to change notification texts like e.g. messenger does when you call somebody. If i understand how it works now it is not possible to change it when app is running.

Reply:
1. You can't because foreground service is bound to the manager and it keeps track of it.
2. This would require more work and can be done as a separate task

Reply:
Create task for it, please. From me green light to merging!